modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java

```diff
@@ -124,6 +114,36 @@ public OAuthConsumerAppDTO createApplication() throws Exception {
 		return createApplication(appDTO);
 	}
 
+	/**
+	 * To set ServiceProvider Provider Claim configuration.
+	 *
+	 * @param serviceProvider Specific Service Provider.
+	 * @return Relevant service provider with updated claim configurations.
+	 */
+	public ServiceProvider setServiceProviderClaimConfig(ServiceProvider serviceProvider) {
+
+		ClaimConfig claimConfig = new ClaimConfig();
```

Comment:
Seems to be tabs used instead of 4 spaces.

Reply:
Fixed with 
8f627c9